### ISSUE TITLE
cant submit hello world on exercism.io

### ISSUE THREAD

**Issue by Mominnaim** [exercism/cli#936]
Hey guys, Im really new to this and I watched every video on youtube on trying to turn in things on Exercism.io and and nothings working. What should I do?

**Comment by kotp** [comment]
Thank you for stopping in to report a problem @Mominnaim.

We will need to know what you have tried in order to help out, I think.

For example, if I were doing a command that failed, I might show this, here:

```sh
exercism --version
Error: unknown flag: --version
```

Then we could give feedback similar to:

> There are no dashes required to get the version of the exercism CLI

**Comment by mustard-studios** [comment]
`Julie-Heths-MacBook-Air:hello-world mstrd-eats$ exercism submit javacript/hello-world/hello-world.js
Error: 

    The file you are trying to submit cannot be found.

        /Users/mstrd-eats/exercism/javascript/hello-world/javacript/hello-world/hello-world.js

        
Julie-Heths-MacBook-Air:hello-world mstrd-eats$ exercism version
exercism version 3.0.13`

This is where I'm at.. I'm thinking of deleting exercism and everything to start over. Thoughts? This is super frustrating

**Comment by mustard-studios** [comment]
I've tried several routes/to/path and none of them work. I've read all of these questions and watched videos on how to properly route my submit and nothing is working. I quit. Just use Codecademy like a normal fucking person.

**Comment by Mominnaim** [comment]
Well imma i don’t know how you got my email, but I’m happy to help,  since
I would need to see how your files are set up and everything, but one
advice I could give is make sure the letter are capped as they should be,
if it is, we should zoom session, but i was mad as hell too but i got it
too work. So lmk if your down to zoom?

**Comment by mustard-studios** [comment]
Ha, I was so angry. I just put a comment in a pull request through the github issues. Sorry if it freaked you out about the email thing. I’m not that good of a hacker. 

To resolve my issue I had to delete everything and start over. My VSC didn’t realize I moved my folder so I just scrapped it and started over. It works and I’m chugging along now.
Thank you for the kindness!!!

Stay safe out there and best regards to you and yours!

Jason Mustard
